### PR TITLE
config: init sync for pod-cost-del-ctl

### PR DESCRIPTION
**Pull request by szuecs**
config: init sync for pod-cost-del-ctl

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -220,6 +220,9 @@ skipper_pod_deletion_cost_controller_memory: 200Mi
 skipper_pod_deletion_cost_controller_cpu: 50m
 # klog style -v=0
 skipper_pod_deletion_cost_controller_log_v: "0"
+# pod informer sync config
+skipper_pod_deletion_cost_controller_poll_interval: "10s"
```
Comment (RomanZavodskikh) [inline cluster/config-defaults.yaml:224]: As far as I remember, interval should be several times smaller than timeout, shouldn't it?

Comment (szuecs) [reply cluster/config-defaults.yaml:224]: Why?
It's also doesn't matter so much because it happens only at startup and test if initial sync was done.

**Comment by RomanZavodskikh** [comment]
:+1:

**Pull request merged**